test/cql-pytest/test_keyspace.py

```diff
@@ -262,6 +262,16 @@ def test_storage_options_alter_type(cql, scylla_only):
         with pytest.raises(InvalidRequest):
             res = cql.execute(f"ALTER KEYSPACE {keyspace} {ksdef_local}")
 
+
+# Test that initial number of tablets is preserved in describe
+def test_describe_initial_tablets(cql, scylla_only):
+    ksdef = "WITH REPLICATION = { 'class' : 'NetworkTopologyStrategy', 'replication_factor' : '1' } " \
+            "AND TABLETS = { 'initial' : 1 }"
+    with new_test_keyspace(cql, ksdef) as keyspace:
+        desc = cql.execute(f"DESCRIBE KEYSPACE {keyspace}")
+        assert "and tablets = {'initial': 1}" in desc.one().create_statement.lower()
```

Comment:
This test only works if tablets are enabled - "test/cql-pytest/run --vnodes" will fail on it, right?
I think you should skip this test if running without tablets, e.g.,
```python
def test_describe_initial_tablets(cql, scylla_only, has_tablets):
   if !has_tablets:
       pytest.skip("This test only makes sense if Scylla supports tablets").
    ...
```

Reply:
I didn't know we had this option (because test.py doesn't and I run it, not custom run-s), thanks! Fixed